### DISCUSSION TITLE
How to create a CFCs  ft_transfer in USN smart contract?

### DISCUSSION THREAD

**Discussion by fradomnt** [near-daos/astro-ui#1095] [Q&A] [answered]
Hi, everyone!
 i sent all my DAO budget to astroDAO via USN token?
How can i create a simple transfer call to our near account in USN?
Cause we had a issues with contrac in USN via "propose a transfer" and  i´d like to learn how create a CFC for this aplication in USN token.

Type:
Smart Contract Address:
Methodo name:
JSON:
deposit:
tGas:
bond:

Thanks a lot!
if i learn i can share in community!
fraDAO

**Comment by fradomnt** [comment]
I used tgas, 150, 180 and nothing.
follow the cfc with error:
![Screen Shot 2022-05-19 at 13 54 48](https://user-images.githubusercontent.com/22111329/169356016-c8b9aa52-b800-4fe6-8428-70ef282a18ea.png)

**Comment by starpause** [comment, marked as answer]
It's now possible to transfer USN with a regular old transfer proposal:

![image](https://user-images.githubusercontent.com/408953/174470157-1dcb0636-57a8-4508-b6b7-1e92bb023623.png)
